cristin-import/src/main/java/no/unit/nva/cristin/lambda/CristinEntriesEventEmitter.java

```diff
@@ -62,18 +62,24 @@ protected String processInput(ImportRequest input, AwsEventBridgeEvent<ImportReq
         S3Driver s3Driver = new S3Driver(s3Client, input.extractBucketFromS3Location());
         String content = fetchFileFromS3(input, s3Driver);
         List<JsonNode> contents = parseContents(content);
-        EventEmitter<JsonNode> eventEmitter = newEventEmitter(context);
-        eventEmitter.addEvents(contents);
-        List<PutEventsResult> failedEntries = eventEmitter.emitEvents();
+        Stream<FileContentsEvent<JsonNode>> eventBodies = generateEventBodies(input, contents);
```

Comment:
This handler does not need to know the specific structure of the the file contents.